util/build_docs.py

```diff
@@ -352,19 +352,25 @@ def install_hugo(install_dir):
     install_dir is created if it doesn't exist yet.
 
     Limitations:
-      Currently only 64 bit Linux is supported."""
+      Currently only 64-bit x86 Linux and macOS is supported."""
 
     # TODO: Support more configurations
-    if platform.system() != 'Linux' or platform.machine() != 'x86_64':
+    if platform.system() == 'Linux' and platform.machine() == 'x86_64':
+        download_url = ('https://github.com/gohugoio/hugo/releases/download/v{version}'
+                '/hugo_extended_{version}_Linux-64bit.tar.gz').format(
+                        version=HUGO_EXTENDED_VERSION)
+
+    if platform.system() == 'Darwin' and platform.machine() == 'x86_64':
+        download_url =  ('https://github.com/gohugoio/hugo/releases/download/v{version}'
+                '/hugo_extended_{version}_macOS-64bit.tar.gz').format(
+                        version=HUGO_EXTENDED_VERSION)
+
+    else:
         logging.fatal(
             "Auto-install of hugo only supported for 64 bit Linux "
```

Comment:
Might want to adjust this error message as well.

Reply:
Done